server/controllers/profileController.js

```diff
@@ -0,0 +1,70 @@
+const Profile = require("../models/ProfileModel");
```

Comment:
This code appears to be from a previous PR. Please rebase your PRs if necessary to only show changes relevant to this PR :)